integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/InsertIntoCarbonTableTestCase.scala

```diff
@@ -252,13 +231,12 @@ class InsertIntoCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
     sql("insert overwrite table CarbonOverwrite select * from THive")
     sql("insert overwrite table HiveOverwrite select * from THive")
     checkAnswer(sql("select count(*) from CarbonOverwrite"), sql("select count(*) from HiveOverwrite"))
-    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, timeStampPropOrig)
+    val folder = new File(s"$storeLocation/default/CarbonOverwrite/Fact/Part0/")
```

Comment:
CarbonOverwrite is always writes in lower case as folder, so use as below to avoid test failure, 
```
val folder = new File(s"$storeLocation/default/carbonoverwrite/Fact/Part0/")
```